### ISSUE TITLE
File not visible in GDrive

### ISSUE THREAD

**Issue by justinpombrio** [brownplt/code.pyret.org#242]
tldr: I just witnessed the version of a file edited in code.pyret.org get out of sync with the version of the file viewable in GDrive.

----

A student was working on a project with two files, one of which imported the other.

At some point, the importing file started seeing an old version of the imported file. Re-running, re-saving, and refreshing the pages of both files had no effect. Viewing the imported file on GDrive showed the old version.

After deleting the imported file in GDrive, the importing file said "file not found", but the imported file could still be edited, saved, refreshed etc. in code.pryet.org.

Using save-as on the imported file fixed everything: the saved-as version (of the same name) was fully visible, both to the importing file and in GDrive.

----

I don't know if this was a problem with code.pyret.org or some weird GDrive issue.

**Comment by schanzer** [comment]
So much has changed in the last 8 (!) years. I believe this is safe to close, as no additional issues have cropped up and this has received no attention.
